70/UnitTest1.cs

```diff
@@ -0,0 +1,22 @@
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace _70
+{
+    public class Tests
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void Test1() {
+            IWebDriver driver = new ChromeDriver();
+            
+            LoginPage loginPage = new LoginPage(driver);
+            loginPage.login("viktoriayaroshenko1@yandex.ru", "Polinka9999");
+        }
```

Comment:
Add any assert method to be sure that your login is really completed successfully.